examples/README.md

```diff
@@ -19,7 +19,7 @@ mvn install -DskipTests
 
 4. Extract REPL/Examples:
 ```
-unzip spark-riak-connector-examples/target/spark-riak-connector-examples-1.0.0-REPL.zip
+unzip examples/target/spark-riak-connector-examples-1.0.0-REPL.zip
```

Comment:
File name was changed to not include version. Should be:
examples/target/spark-riak-connector-examples-REPL.zip